docs/migrations.md

```diff
@@ -29,9 +30,9 @@ into the concepts presented throughout this page:
 ## Schema migrations
 
 Any time you add or change a field on a Django model, Wagtail page model
-(which are a particular kind of Django model), or StreamField block classes, a
+(which are a particular kind of Django model), or StreamField block class, a
 [Django schema migration](https://docs.djangoproject.com/en/1.11/topics/migrations)
-will be required.
+will be required. This includes changes as small as modifying the `help_text` string.
```

Comment:
I think this is the simplest way to make the point, based on both @chosak and @Scotchester's suggestions.